_docs_v5/4_guides/scripting.md

```diff
@@ -246,23 +246,130 @@ Next is a tool bar that displays the currently executing script and three button
+| get_output_logs_filenames | Command and Telemetry Server | Deprecated |
+| get_packet_data | Command and Telemetry Server | Deprecated |
+| get_packet_logger_info | Command and Telemetry Server | Deprecated |
+| get_packet_loggers | Command and Telemetry Server | Deprecated |
+| get_replay_mode | Replay | Deprecated |
+| get_router_info | Command and Telemetry Server | Deprecated, use get_router |
+| get_screen_definition | Telemetry Viewer | Unimplemented |
+| get_screen_list | Telemetry Viewer | Unimplemented |
+| get_scriptrunner_message_log_filename | Command and Telemetry Server | Deprecated |
+| get_server_message | Command and Telemetry Server | Deprecated |
+| get_server_message_log_filename | Command and Telemetry Server | Deprecated |
+| get_server_status | Command and Telemetry Server | Unimplemented |
+| get_target_ignored_items | Command and Telemetry Server | Deprecated, use get_target |
+| get_target_ignored_parameters | Command and Telemetry Server | Deprecated, use get_target |
+| get_target_info | Command and Telemetry Server | Deprecated, use get_target |
+| get_tlm_details | Command and Telemetry Server | Deprecated |
+| get_tlm_item_list | Command and Telemetry Server | Deprecated |
+| get_tlm_list | Command and Telemetry Server | Deprecated |
+| get_tlm_log_filename | Command and Telemetry Server | Deprecated |
+| interface_state | Command and Telemetry Server | Deprecated, use get_interface |
+| local_screen | Script Runner | Unimplemented |
+| map_target_to_interface | Command and Telemetry Server | Unimplemented |
+| override_tlm_raw  | Command and Telemetry Server | Deprecated, use override_tlm |
+| open_file_dialog | Script Runner | Unimplemented |
+| open_files_dialog | Script Runner | Unimplemented |
+| open_directory_dialog | Script Runner | Unimplemented |
+| prompt_combo_box | Script Runner | Unimplemented |
+| prompt_message_box | Script Runner | Unimplemented |
+| prompt_to_continue | Script Runner | Unimplemented |
+| prompt_vertical_message_box | Script Runner | Unimplemented |
+| replay_move_end | Replay | Deprecated |
+| replay_move_index | Replay | Deprecated |
+| replay_move_start | Replay | Deprecated |
+| replay_play | Replay | Deprecated |
+| replay_reverse_play | Replay | Deprecated |
+| replay_select_file | Replay | Deprecated |
+| replay_set_playback_delay | Replay | Deprecated |
+| replay_status | Replay | Deprecated |
+| replay_step_back | Replay | Deprecated |
+| replay_step_forward | Replay | Deprecated |
+| replay_stop | Replay | Deprecated |
+| router_state | Command and Telemetry Server | Deprecated, use get_router |
+| run_mode | Script Runner | Unimplemented |
+| save_file_dialog | Script Runner | Unimplemented |
+| script_disconnect | Script Runner | Unimplemented |
+| set_disconnected_targets | Script Runner | Unimplemented |
+| set_replay_mode | Replay | Deprecated |
+| set_stdout_max_lines | Script Runner | Deprecated |
```

Comment:
I don't remember this, but might still want?  I think this clipped stdout at some size to prevent lockups

Reply:
I'll mark as unimplemented then